nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/BasePermissionsJsonEnricherTest.java

```diff
@@ -44,12 +46,12 @@ public BasePermissionsJsonEnricherTest() {
 
     @Test
     public void test() throws Exception {
+        PermissionProvider permissionProvider = Framework.getService(PermissionProvider.class);
         DocumentModel root = session.getDocument(new PathRef("/"));
         JsonAssert json = jsonAssert(root, CtxBuilder.enrichDoc("permissions").get());
         json = json.has("contextParameters").isObject();
         json.properties(1);
-        json = json.has("permissions").contains("Read", "Write", "Everything",
-            "AddChildren", "ReadChildren", "RemoveChildren", "ReadWrite");
+        json = json.has("permissions").contains(permissionProvider.getPermissions());
```

Comment:
This doesn't really test much more than permissionProvider.getPermissions() (and was a problem with the pre-existing test also).

What we really want to test is access to a document with a specific ACL by a non-system user, to see that only the appropriate permissions are returned, and not all of them.